health-services/ExerciseSampleCompose/gradle/libs.versions.toml

```diff
@@ -12,6 +12,7 @@ org-jetbrains-kotlin = "1.9.25"
 ui-test-junit4 = "1.6.8"
 robolectric = "4.13"
 roborazzi = "1.26.0"
+runtime-livedata = "1.6.8"
```

Comment:
Do you really need livedata?

Reply:
Builds without it! Must've been something I was playing around with and never deleted the dependencies for 👍🏾